src/awkward/operations/ak_from_parquet.py

```diff
@@ -220,13 +220,15 @@ def _load(
 
     if len(arrays) == 0:
         return wrap_layout(
-            subform.length_zero_array(highlevel=False), behavior=behavior
+            subform.length_zero_array(highlevel=False),
+            highlevel=highlevel,
+            behavior=behavior,
```

Comment:
It doesn't look like we can actually trigger this pathway - the `metadata` function validates the path list to ensure it's non-empty. We should perhaps remove this case from the function.